flamingpy/utils/viz.py

```diff
@@ -284,6 +285,138 @@ def draw_EGraph(
 
     return fig, ax
 
+@mpl.rc_context(plot_params)
+def draw_EGraph_3DScatterPlot(
```

Comment:
@sduquemesa do you have any preference on how we will call this plot method? Should we use seperate functions/methods or should we pass "plotly" as kwarg to the EGraph draw method?

Reply:
I would be in favour of renaming this function 
```suggestion
def draw_EGraph_plotly(
```
and then add a key word argument to the draw method of `EGraph`. Then `EGraph.draw` would look something like
```
    def draw(self, backend='matplotlib', **kwargs):
        """Draw the graph state with Matplotlib.

        See flamingpy.utils.viz.draw_EGraph and flamingpy.utils.viz.draw_EGraph_plotly for more details.
        """
        if backend == 'matplotlib':
                from flamingpy.utils.viz import draw_EGraph

                fig, ax = draw_EGraph(self, **kwargs)
                return fig, ax
        elif backend == 'plotly':
                from flamingpy.utils.viz import draw_EGraph_plotly
                
                fig = draw_EGraph_plotly(self, (**kwargs)
                return fig
```
What do you think @Avhijit-codeboy? Can you implement this?

Reply:
@soosub, I like it! What about something like this? just to remove redundant code

```
    def draw(self, backend='matplotlib', **kwargs):
        """Draw the graph state with Matplotlib.

        See flamingpy.utils.viz.draw_EGraph and flamingpy.utils.viz.draw_EGraph_plotly for more details.
        """
        if backend == 'matplotlib':
            from flamingpy.utils.viz import draw_EGraph
        elif backend == 'plotly':
            from flamingpy.utils.viz import draw_EGraph_plotly as draw_EGraph

        fig, ax = draw_EGraph(self, **kwargs)
        return fig, ax
```

By the way, @Avhijit-codeboy are this plots producing the expected output already?

Reply:
Hi @sduquemesa . This is absolutely implementable.
I haven't yet gone through the changes to see whether we are getting the expected output or not. Will do so shortly.

Reply:
The nodes are properly connected now.![image](https://user-images.githubusercontent.com/41890388/175784661-fcfeb317-238e-4b3e-b161-fb9d03852027.png)